sdk/synapse/Azure.Analytics.Synapse.Spark/tests/SparkSessionClientLiveTests.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Analytics.Synapse.Spark.Models;
+using Newtonsoft.Json.Linq;
```

Comment:
Nit - can you use System.Text.Json instead?

Reply:
Fixed